package.json

```diff
@@ -51,22 +51,25 @@
     "sls": "./bin/serverless.js"
   },
   "scripts": {
+    "commitlint": "commitlint -f HEAD@{15}",
```

Comment:
@pmuens and @medikoo This comment is not related to this PR.. My changes which were supposed to go in 1.59.1 were not included.

Since this looks like next release I wanted to bring above to your notice. Please make sure https://github.com/serverless/serverless/pull/7040 changes are packaged into next release..

If you're wondering what happend here are the logs.

https://travis-ci.org/serverless/serverless/builds/621107104?utm_source=github_status&utm_medium=notification

https://travis-ci.org/serverless/serverless/jobs/621107105?utm_medium=notification&utm_source=github_status

Thanks for your help.